R/plumb-block.R

```diff
@@ -279,10 +286,18 @@ evaluateBlock <- function(srcref, file, expr, envir, addEndpoint, addFilter, mou
     }
 
     stat <- PlumberStatic$new(block$assets$dir, expr)
-    mount(path, stat)
-
+    pr$mount(path, stat)
+
+  } else if (!is.null(block$routerModifier)) {
+    if (is.expression(expr)){
+      func <- eval(expr, envir)
+      if (is.function(func)) {
+        func(pr)
+        return()
+      }
+    }
+    message("Invalid expression for @plumber tag. Use form function(pr) {pr$...}.")
```

Comment:
Can we move this block up to be checked first? I'd like it to have first preference.

Inside this block, should we check for other fields? Ex: `paths` should be NULL, `filter` should be NULL, etc

Reply:
The other sections do not check for other fields. I'm ok duplicating this behavior

Reply:
Now, I see the check above. 🤦

Reply:
If all other sections are missing, then order does not matter.

Closing comment / suggestion

Reply:
There is a check above, they can not all exist at the same time.